packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,311 @@
+import * as React from 'react';
+import { useState } from 'react';
+import styles from '@patternfly/react-styles/css/components/Slider/slider';
+import { css } from '@patternfly/react-styles';
+import { SliderStep } from './SliderStep';
+import { InputGroup, InputGroupText } from '../InputGroup';
+import { TextInput } from '../TextInput';
+
+export interface SliderStepObject {
+  /** Value of the step. This value is a percentage of the slider where the  tick is drawn. */
+  value: number;
+  /** The display label for the step value. THis is also used for the aria-valuetext */
+  label: string;
+  /** Flag to hide the label */
+  isLabelHidden?: boolean;
+}
+
+export interface SliderProps extends Omit<React.HTMLProps<HTMLDivElement>, 'onChange'> {
+  /** Additional classes added to the spinner. */
+  className?: string;
+  /** Current value  */
+  currentValue?: number;
+  /** Flag indicating if the slider is is discrete */
+  isDiscrete?: boolean;
+  /** Array of slider step objects (value and label of each step) for the slider. If this is provided, the numSteps prop will be ignored.*/
```

Comment:
is there still a `numSteps` prop? I dont see it.

Reply:
nope. that was removed